web/src/i18n/locales/ja.json

```diff
@@ -34,18 +34,18 @@
     },
     "faq": {
         "correctionA": "ご質問がございましたら、下記までお問い合わせください。\n\nYan Fan [英語]\nyan@codechrysalis.io\n\nEmi Takemura Miller [日本語]\nemitakemura@gmail.com",
-        "correctionQ": "私は指名されましたが、私の情報を修正または削除してもらいたいです。",
-        "freeA": "このWebサイトを使用してスピーカーに連絡することは無料ですが、一部のスピーカーはサービスに課金する場合があり、他のスピーカーは無料で話すことを選択する場合があります。メンバーの価値を設定し、時間とエネルギーを寄付する方法を決定するというメンバーの決定をサポートします。",
-        "freeQ": "スピーカーは無料でプレゼンテーションを行いますか？",
-        "obligationA": "スピーカーが私たちのプラットフォームにリストされている場合でも、彼らはプレゼンテーションをしたり、メッセージに応答したりする義務はありません。私たちのサービスは、イベント主催者と女性スピーカーをつなぐことを目的としています。そこから、両方の当事者が機会が適切かどうかを判断できます。",
-        "obligationQ": "講演者は講演する義務がありますか？",
+        "correctionQ": "リストに掲載されている私の情報を修正または削除してもらいたいです。",
+        "freeA": "このWebサイトを通じてスピーカーに連絡することは無料で可能ですが、中には有料のスピーカーもいれば、無料で講演を行うスピーカーもいます。私たちは、メンバーの価値を設定し、時間とエネルギーを寄与する方法をサポートします。",
+        "freeQ": "スピーカーは無料で講演を行ってくれますか？",
+        "obligationA": "スピーカーが私たちのプラットフォームに登録されている場合でも、彼らは講演をしたり、メッセージに応答したりする義務はありません。私たちのサービスは、イベント主催者と女性スピーカーをつなぐことを目的としています。そこから、両当事者で機会が合っているかどうかを判断してください。",
+        "obligationQ": "スピーカーは講演する義務がありますか？",
         "qualifyA": "人前で話すための最大のハードルは、しばしば私たち自身の恐れです。私たちは、トレーニングや経験に関係なく、人前で話すことに関心がある女性なら誰でも自分を推薦することを歓迎します。",
         "qualifyQ": "スピーカーになるための前提条件はありますか？",
         "title": "FAQ",
-        "whoA": "これは、Yan Fan、キンテラ　トゥッチ、キルざー杏によって作成されたオープンソースプロジェクトです。オリジナルのアイデアは、竹村えみミラーさんからです。更なる支援は、Tokyo Women Codeによって提供されました。寄付をしたい場合は、GitHubページをチェックしてください。",
+        "whoA": "これは、Yan Fan、Tutti Quintella、Ann Kilzerらによって作成されたオープンソースプロジェクトです。オリジナルのアイデアは、Emi Takemura Millerさんからです。更なる支援は、Tokyo Women Codeによって提供されました。参加したい場合は、GitHubページをチェックしてください。",
```

Comment:
Can we leave the names in Kana/Kanji? キンテラトゥッチ、キルザー杏　and 竹村えみミラーさん?

Reply:
I'd rather have my name in English, cause if you google キンテラトゥッチ you won't find leads to me on the internet. Tutti Quintella is the way to find me.

Reply:
Makes sense. For SEO, the site has an english version too, though I wonder what robots will see 🤔

Reply:
If this name has an official name in Japanese, I will revise it to that.
Otherwise, I felt there was no point in forcing it into Japanese.
If correct it, please let me know the official name in Japanese :)

Reply:
Emi’s name should be in Japanese. For my name, either way is okay. Perhaps it’s more consistent to use English. Thanks!